api/src/main/scala/org/ensime/api/incoming.scala

```diff
@@ -56,6 +56,11 @@ final case class TypecheckFileReq(fileInfo: SourceFileInfo) extends RpcAnalyserR
 final case class UnloadModuleReq(module: String) extends RpcAnalyserRequest
 
 /**
+ * Responds with a `VoidResponse`
+ */
+final case class UnloadFileReq(file: File) extends RpcAnalyserRequest
```

Comment:
Would it make sense to use URIs to talk about file identity when the contents don't matter?